### ISSUE TITLE
NTR: Tripeptidyl peptidase enzyme activity low in leukocytes and fibroblasts (Lab: blood chemistry: enzymes: leukocyte)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8139]
New term request Tripeptidyl peptidase enzyme activity low in leukocytes and fibroblasts (forsyt_070802080255): TPP

**Comment by pnrobinson** [comment]
@MickeySegal -- there are two versions of this enzyme. It seems TPP1 is more important clinically, I am using that

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000571
- label: Reduced tissue tripeptidyl peptidase 1 activity
- parents: [Abnormal tissue enzyme activity (HP:6000231)](https://hpo.jax.org/app/browse/term/HP:6000231)
- pmid: PMID:37922835
- def: Activity of tripeptidyl peptidase 1 (TPP1;EC 3.4.14.9) in the tissues below the lower limit of normal. TTP1 activity can be measured in tissues including fibroblasts and leukocytes muscle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8139
- synonyms: Reduced leukocyte tripeptidyl peptidase 1 activity(NONE)|Reduced tripeptidyl peptidase 1 activity in cultured fibroblasts(NONE)
